src/component/axisPointer/modelHelper.ts

```diff
@@ -194,8 +196,9 @@ function collectAxesInfo(result: CollectionResult, ecModel: GlobalModel, api: Ex
                 : axisPointerModel;
 
             const snap = axisPointerModel.get('snap');
+            const triggerEmphasis = axisPointerModel.get('triggerEmphasis');
             const axisKey = makeKey(axis.model);
-            const involveSeries = triggerTooltip || snap || axis.type === 'category';
+            const involveSeries = triggerTooltip || triggerEmphasis || snap || axis.type === 'category';
```

Comment:
I _think_ the addition of `triggerEmphasis` here might be causing the unexpected visual diffs. Removing it and testing it out locally to validate.